### PR TITLE
dbnet nntc(1684x fp32&int8) &mlir(1684x&1684 fp32)

### PR DESCRIPTION
**Pull request by seriphdog**
###submit to github
1) has add to full_cases.txt
2) Pr Info:
2.1) tpu-mlir bm1684x performance in stat.csv:
name,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
dbnet_bm1684x_Bf16,2x3x736x1280,5.255,22.638,11.41%,2.00%,35.18%
dbnet_bm1684x_f16,2x3x736x1280,5.255,22.798,11.53%,2.00%,35.20%
dbnet_bm1684x_f32,2x3x736x1280,5.255,157.368,1.67%,5.00%,12.50%
compilation,2x3x736x1280,5.255,22.808,N/A,2.00%,N/A
compilation,2x3x736x1280,5.255,22.804,N/A,2.00%,N/A
compilation,2x3x736x1280,5.255,22.804,N/A,2.00%,N/A
fp32:flops: 92343593768, runtime: 99.522163ms, ComputationAbility: 0.927870T
fp16:flops: 92343593768, runtime: 19.114132ms, ComputationAbility: 4.831169T
bf16:flops: 65554824720, runtime: 21.997709ms, ComputationAbility: 2.980075T
2.2)tpu-mlir bm1684 performace in stat.csv
name,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
dbnet_bm1684_f32,,5.255,nan,nan%,0.00%,nan%
compilation,,5.255,nan,N/A,0.00%,N/A
fp32:flops: 92343593768, runtime: 138.740356ms, ComputationAbility: 0.665586T
2.2)tpu-mlir:
branch:master
commit id: cb3736f7759f28ba6e91911cfce3a6a8ab52ae1a
2.3)libsophon:0.4.5
2.4)tpu-nntc:
branch:Release
commit id: 315ccc9a5ba3b401039e399fd6f8cdc92f70c78b
2.5) tpu-nntc bm1684x performance in stat.csv:
name,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
fp32:flops: 52233002534, runtime: 57.945896ms, ComputationAbility: 0.901410T
int8:flops: 52254391204, runtime: 7.832151ms, ComputationAbility: 6.671781T

### CODE REVIEW

File: vision/detection/dbnet/data/icdar2015/ch4_quanti_data/img_1.jpg
Comment (Jeffwhen) [inline vision/detection/dbnet/data/icdar2015/ch4_quanti_data/img_1.jpg:1]: 用 LFS 上传。

**Review by Jeffwhen — CHANGES_REQUESTED** [review]
Reference input 能不能通过图片用指定预处理参数的方法来做？

File: vision/detection/dbnet/data/icdar2015/ch4_quanti_data/img_1.jpg
Comment (seriphdog) [reply vision/detection/dbnet/data/icdar2015/ch4_quanti_data/img_1.jpg:1]: 1） input_npz/output_npz已经删除，model_transform可输入图片，自动生成该图对应的Input_npz和output_npz
2）1.jpg已lfs ,"*.jpg"已加入gitignore

**Comment by Jeffwhen** [comment]
参考 [MobileNet-Caffe-v2/mlir.config.yaml](https://github.com/sophgo/model-zoo/blob/main/vision/classification/MobileNet-Caffe-v2/mlir.config.yaml) 改一下，还需要做一下 rebase。

**Comment by seriphdog** [comment]
> 

1)deploy-> +deploy done
2)rebase disasters caused by various others' force push have been eliminated

**Comment by seriphdog** [comment]
image
tpu-nntc重复载入，疑似清空不彻底 @@QinyuZHAO0

**Comment by seriphdog** [comment]
image
非编译阶段异常，疑似84X未清空，请排查

**Comment by QinyuZHAO0** [comment]
正进行本地复现以定位问题



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年05月08日 13:55 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [sophgo/model-zoo] dbnet nntc(1684x fp32&int8) &mlir(1684x&1684 fp32) (PR #200) |


非编译阶段异常，疑似84X未清空，请排查

**Comment by Jeffwhen** [comment]
需要重新 rebase 下，把 facenet 那一笔去掉。

**Comment by seriphdog** [comment]
> 需要重新 rebase 下，把 facenet 那一笔去掉。

Done,Please Check

**Comment by QinyuZHAO0** [comment]
已手动cherry-pick合入

**Pull request closed without merging**